docs/content/configuration.md

```diff
@@ -93,12 +94,14 @@ If the `db-tools:check` command returns you some errors:
       # Specify here paths to binaries, only if the system can't find them by himself
       # platform are 'mysql', 'postgresql', 'sqlite'
       backupper_binaries:
-          postgresql: 'usr/bin/pg_dump' # default 'pg_dump'
+          mariadb: 'usr/bin/mariadb-dump' # default 'mariadb-dump'
```

Comment:
It's weird to see absolute paths without the leading `/`, is it voluntary ?

Reply:
no !